file-upload-tutorial.md

```diff
@@ -0,0 +1,25 @@
+# A tutorial on doing file uploads in Hapi
+
```

Comment:
Personally, I like to know on the first line what is happening in the rest of the blog post. This is a very personal opinion but I would suggest you consider adding an introductory sentence around how as part of week x of the F&C course you put together a tutorial that does x.